README.md

```diff
@@ -1 +1,23 @@
-# stylelint-create
\ No newline at end of file
+# stylelint-create
+
+The official (in-development) [`npm init`](https://docs.npmjs.com/cli/v9/commands/npm-init) script for [Stylelint](https://github.com/stylelint/stylelint).
+
+Tracking issue: https://github.com/stylelint/stylelint/issues/4269
+
+## Use the tool
+
+Coming soon!
+
+## Getting started
+
+- `npm install`
+- to test locally, run `node create-stylelint.mjs`
+- to test with [Vitest](https://vitest.dev/), `npm test`
+
+Commits to `main` will be tested by [GitHub Actions](https://github.com/stylelint/create-stylelint/actions)
+
+## License
+
+This repository is licensed under the MIT License.
+
+The development of this project is heavily inspired by [create-astro](https://github.com/withastro/astro/blob/main/packages/create-astro); thank you to the Astro team!
```

Comment:
As per my above comment on removing the LICENSE file, this is also probably not required as the license will be displayed on the repo as it is for the main Stylelint repo: https://github.com/stylelint/stylelint

![image](https://user-images.githubusercontent.com/1016458/214996420-afaf5b06-6142-43b8-b01f-740189fe4a42.png)